backend/schemas/source/base/ClusterNames.json

```diff
@@ -3,10 +3,8 @@
     "HEAD START CLUSTER",
     "477 CLUSTER",
     "MATERNAL, INFANT, AND EARLY CHILDHOOD HOME VISITING CLUSTER",
-    "NAN",
```

Comment:
Looks like the schema build process was turning `N/A` into `"NAN", which explains why we needed the extra constant in `Base.libsonnet`

Reply:
This is actually a default behavior for pandas: https://pandas.pydata.org/pandas-docs/stable/user_guide/io.html#na-values